server/src/http/streams.rs

```diff
@@ -43,8 +41,8 @@ async fn get_streams(
     Extension(identity): Extension<Identity>,
 ) -> Result<Json<Vec<Stream>>, CustomError> {
     let system = state.system.read().await;
-    system.permissioner.get_streams(identity.user_id)?;
-    let streams = mapper::map_streams(&system.get_streams()).await;
+    let streams = system.find_streams(&Session::stateless(identity.user_id))?;
+    let streams = mapper::map_streams(&streams).await;
```

Comment:
let streams = mapper::map_streams(&system.find_streams(&Session::stateless(identity.user_id))?).await;

Reply:
I did it on purpose, just to make the code a bit easier to read (e.g. the same approach could be applied in `topics` and a few other places).